### PR TITLE
update CMakeLists.txt to 3.5,  fix FindTinyXML.cmake search way and change include way

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

**Comment by phunkyfish** [comment]
Can we change Xcode and add Appveyor here too?

**Comment by AlwinEsch** [comment]
Appveyor yes, but not Xcode, here it use the Version of Kodi Leia

But for this is more @Rechi the chief.

EDIT:
I add Appveyor, Rechi about Versions used there.

**Comment by phunkyfish** [comment]
Makes sense on Xcode version. Will test both changes today and tomorrow once Appveyor is there.

**Comment by phunkyfish** [comment]
This needs the version and changelog added. Is this ready to go in once tested @AlwinEsch ?

**Comment by AlwinEsch** [comment]
If the changelog is good can it in.

File: pvr.vuplus/addon.xml.in [hunk 1]
```diff
@@ -175,6 +175,11 @@
     <disclaimer lang="zh_TW">這是測試版軟體！其原創作者並無法對於以下情況負責，包含：錄影失敗，不正確的定時設定，多餘時數，或任何產生的其它不良影響...</disclaimer>
     <platform>@PLATFORM@</platform>
     <news>
+v3.27.1
```
Comment (phunkyfish) [inline pvr.vuplus/addon.xml.in:178]: Also need this in changelog.txt

**Comment by phunkyfish** [comment]
I just need to test the Leia PR

**Comment by phunkyfish** [comment]
Tested and works as expected.

**Review by phunkyfish — APPROVED** [review]

**Comment by phunkyfish** [comment]
@AlwinEsch is this PR ok to be merged also, i.e. any backport PRs are already in Leia?

**Comment by AlwinEsch** [comment]
Can go in, is mainly to be ready if Kodi also gets Leia cmake 3.5 and thought to test the build anywhere to see any bugs.

**Pull request merged**